### PR TITLE
🐛 Fix md syntax in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dev-drprasad**
# What does this change
Just md syntax fix
# What issue does it fix
N/A

[1]: /CONTRIBUTING.md#when-to-open-a-pull-request

# Notes for the reviewer

# Checklist
- [x] Unit Tests
- [x] Documentation
- [x] Schema (porter.yaml)

[contributors]: /CONTRIBUTORS.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -344,7 +344,7 @@ a new blog post and then preview it:
 1. [Preview](#preview-documentation) the website and click "Blog" at the top right to find your blog post.
 
 1. When you create a pull request, look at the checks run by the pull request,
-    and click "Details" on the *netlify/porter/deploy-preview** one to see a live
+    and click "Details" on the **netlify/porter/deploy-preview** one to see a live
```
Comment (dev-drprasad) [inline CONTRIBUTING.md:347]: It's still unclear for me whether it was supposed to be *italic* or **bold** :)

**Comment by dev-drprasad** [comment]
https://twitter.com/rafrasenberg/status/1311627619973107715

This is totally unnecessary comment and I don't why I am posting here

**Review by carolynvs — APPROVED** [review]
LGTM, thank you! 💖

**Comment by carolynvs** [comment]
Oops, the [DCO check](https://github.com/getporter/porter/blob/main/CONTRIBUTING.md#signing-your-commits) is failing. You can fix it by running `git commit --amend -s` then force pushing back up to your branch.

**Comment by dev-drprasad** [comment]
@carolynvs re-committed

**Pull request merged**